src/scopmath/scoplib.h

```diff
@@ -1,145 +1,188 @@
-				 * coef,linflag) */
+int _advance(int ninits,
+             int n,
+             int* s,
+             int* d,
+             double* p,
+             double* t,
+             double dt,
+             int (*fun)(),
+             double*** pcoef,
+             int linflag);
 
 /* Solution of boundary value problems */
-
-int 
-boundary(),			/* (npts, x, y, p, fval, gval, qval) */
-shoot();			/* (nbound, indepindex, matchtime, matchvalue,
-				 *  known, unknown, p, diff) */
+int boundary(int npts, double* x, double* y, double (*f)(), double (*g)(), double (*q)());
 
 /* Solution of parabolic partial differential equations */
-
-int crank();			/* (n, y, fval, gval, p, dt, dx, first_time,
-				 * bound_string) */
+int crank(int n,
+          double* y,
+          double* fval,
+          double* gval,
+          double dt,
+          double dx,
+          double t,
+          double** bound,
+          double** pwork);
 
 /* Definite integrals */
-
-int romberg();			/* (a, b, func, p, integral) */
-double legendre();		/* (a, b, func, p) */
+double romberg(double a, double b, int (*func)());
+double legendre(double a, double b, int (*func)());
 
 /* Solution of simultaneous algebraic equations */
-
-int 
-simeq(), seidel(),		/* (n, coef, &soln) */
-invert(),			/* (n, matrix) */
-crout(),			/* (n, matrix, perm) */
-solve(),			/* (n, matrix, const_vect, perm, &soln) */
-tridiag(),			/* (n, super, diag, sub, const, &soln) */
-newton(),			/* (n, &soln, p, funcval, &value) */
-simplex(),			/* (n, &soln, p, funcval, &value) */
-buildjacobian();		/* (n, &var, p, funcval, value, jacobian) */
-
-/*  Sensitivity analysis */
-
-int 
-linearsens(),			/* (nvar,&var,p,param,funcval,coeff,newjac,
-				 * &sens) */
-steadysens(),			/* (nvar,&var,p,param,funcval,value,newjac,
-				 * &sens) */
-trajecsens(),			/* (nvar,&var,&der,param,first_time,newjac,
-				 * &sens,&Dsens) */
-envelope();			/* (&var, vsize, param, uncert, &sens, &plus,
-				 * &minus) */
+int simeq(int n, double** coef, double* soln, int* index);
+int seidel(int n, double** coef, double* soln, int* index);
+int invert(int n, double** matrix);
+int crout(int n, double** a, int* perm);
+int solve(int n, double** a, double** b, int* perm, double* p, int* y);
+int tridiag(int n, double* a, double* b, double* c, double* d, double* soln);
+int newton(int n, int* index, double* x, int (*pfunc)(), double* value);
+int simplex(int nparms, int* parms, double* pp, int (*pfunc)(), int* value);
+int buildjacobian(int n, int* index, double* x, int (*pfunc)(), double* value, double** jacobian);
 
 /* Curve-fitting and interpolation functions */
-
-int derivs();			/* (nbase, x, y, mesh, der) */
-double spline();		/* (nbase, x, y, mesh, der, x_inter) */
-double force();			/* (t, filename) */
-double stepforce();			/* (..., t, filename) */
-#define deflate scoplib_deflate /* causes python gzip to fail if nrniv launched. i.e zlib declares global deflate */
```

Comment:
I missed this comment, and this seems to be the source of the documentation CI issue.

Reply:
Re-added in 5ef8ed7ab9c8cd4a6eb448eb6c97837c8362f69e with an updated comment.